app/records-interfaces/allocation-records-interface.coffee

```diff
@@ -0,0 +1,13 @@
+null 
+
+### @ngInject ###
+global.cobudgetApp.factory 'AllocationRecordsInterface', (config, BaseRecordsInterface, AllocationModel) -> 
+  class AllocationRecordsInterface extends BaseRecordsInterface
+    model: AllocationModel
+    constructor: (recordStore) ->
+      @baseConstructor recordStore
+      @restfulClient.apiPrefix = config.apiPrefix 
```

Comment:
i think you might need to only do this once in https://github.com/cobudget/cobudget-ui/blob/beta/app/angular-record-store.coffee.